Plots/XY/NCL_xy_13.py

```diff
@@ -0,0 +1,79 @@
+"""
+NCL_xy_12.py
+===============
+This script illustrates the following concepts:
+   - Adding error bars on an XY plot
```

Comment:
I think this needs to be indented one space

Reply:
Good catch! I just fixed it